src/controllers/simplified-account/services/payment-links/create/payment-link-review.controller.ts

```diff
@@ -90,11 +91,10 @@ async function post(req: ServiceRequest, res: ServiceResponse) {
     .withType(productTypes.ADHOC)
 
   const paymentLink = await createProduct(createProductRequest)
-  const goLiveLink = formatServicePathsFor(paths.service.requestToGoLive.index, req.service.externalId) as string
-  const successBannerBody = `You can <a href="${paymentLink.links.pay.href}/"
-    class="govuk-link govuk-link--no-visited-state" target="_blank">test your payment link</a>.
-    You'll need to create it again if you want to use it when you <a href="${goLiveLink}"
-    class="govuk-link govuk-link--no-visited-state" target="_blank">go live</a>.`
+  const successBannerBody = (service.currentGoLiveStage !== goLiveStage.LIVE || account.paymentProvider.toLowerCase() === PaymentProviders.SANDBOX ) ?
```

Comment:
I think you just want to look at the GatewayAccount type here instead, sandbox mode can be all three types of payment provider

Reply:
Thanks!

Reply:
Amended, thanks. Ready for re-review when you can.